var/spack/repos/builtin/packages/resolve/package.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Resolve(CMakePackage, CudaPackage):
+    """ReSolve is a library of GPU-resident sparse linear solvers. It contains iterative and direct solvers designed to run on NVIDIA and AMD GPUs, as well as CPU devices."""
+
+    homepage = "https://code.ornl.gov/ecpcitest/exasgd/resolve"
+    url = "https://code.ornl.gov/ecpcitest/exasgd/resolve.git"
```

Comment:
```suggestion
    homepage = "https://github.com/ORNL/ReSolve"
    url = "https://github.com/ORNL/ReSolve"
```
I would just use URL where Re::Solve will be published in anticipation this will happen soon. :)

Reply:
Perhaps we can just leave this as a draft until we have a public URL? If we want to merge, I would rather have a functioning, private repo than have to edit the link to use the package.

Reply:
Agreed! We can change this once we move to GitHub.

Reply:
Now using GitHub URL